Source/IGListKit/Internal/IGListAdapter+UICollectionView.m

```diff
@@ -235,6 +235,19 @@ - (void)collectionView:(UICollectionView *)collectionView didUnhighlightItemAtIn
     [sectionController didUnhighlightItemAtIndex:indexPath.item];
 }
 
+#if !TARGET_OS_TV
+- (UIContextMenuConfiguration *)collectionView:(UICollectionView *)collectionView contextMenuConfigurationForItemAtIndexPath:(NSIndexPath *)indexPath point:(CGPoint)point API_AVAILABLE(ios(13.0)) {
+  // forward this method to the delegate b/c this implementation will steal the message from the proxy
+  id<UICollectionViewDelegate> collectionViewDelegate = self.collectionViewDelegate;
+  if ([collectionViewDelegate respondsToSelector:@selector(collectionView:contextMenuConfigurationForItemAtIndexPath:point:)]) {
+      [collectionViewDelegate collectionView:collectionView contextMenuConfigurationForItemAtIndexPath:indexPath point:point];
```

Comment:
I think here should be `return [collectionViewDelegate collectionView:collectionView contextMenuConfigurationForItemAtIndexPath:indexPath point:point];`

Reply:
Nope it's a call to the delegate. 
I don't see anything blocking in this PR, apart from the fact it's getting old now :)

Reply:
My 2 cents: I too think this line is missing a `return` in the beginning. As far as I'm aware `collectionView:contextMenuConfigurationForItemAtIndexPath:point:`'s purpose is to **return** a context menu configuration, not simply to notify the delegate (see [docs](https://developer.apple.com/documentation/uikit/uicollectionviewdelegate/3295917-collectionview)).

If you do `return` here you would give the developer option to either add context menu configuration to `collectionViewDelegate` or to `sectionController`. If you do not return here there's no reason to implement this method in `collectionViewDelegate` since it wouldn't do anything.

At least I would have been surprised if I had implemented `collectionView:contextMenuConfigurationForItemAtIndexPath:point:` in `collectionViewDelegate`, returned proper configuration and it wouldn't cause context menu to show up.